### ISSUE TITLE
Edit Profile in UserProfile

### ISSUE THREAD

**Issue by aitikgupta** [MLH-Fellowship/SocioMark#84]
We can reuse `CreatePost` with the right handler to Edit a user's profile Bio and Profile Picture.
Currently #82 does not implement this yet.

**Comment by sameeerkashyap** [comment]
Hey!, me and @suhanichawla would like to work on this. Please assign us.

**Comment by aitikgupta** [comment]
Sure @Sameerkash!
Relevant entrypoint would be https://github.com/MLH-Fellowship/SocioMark/blob/main/frontend/src/Components/UserProfile.js

And this specific line: https://github.com/MLH-Fellowship/SocioMark/blob/8ecdf24ab14f09a1daa2ea5eb99901624e6057cc/frontend/src/Components/UserFeed.js#L53

**Comment by sameeerkashyap** [comment]
@aitikgupta  Can we use this endpint? `@router.patch("/update", response_description="Update user from the database")`

**Comment by sameeerkashyap** [comment]
And we're a bit stuck with the styling, what resources should we use? we're guessing its tailwind, but not entirely sure as we don't see any CSS. Could you help? @aitikgupta

**Comment by aitikgupta** [comment]
Yes, it's Tailwind! Their [docs](https://tailwindcss.com/docs) are pretty extensive I believe.

Ping @bodhisha, could you send some useful links?

**Comment by bodhisha** [comment]
@Sameerkash  Yea.. Its same as normal css just that we use shortcuts inside the classname="" of div. Say, if we need to add padding to a div just add classname="p-4". Similarly everything works.. Their official documentation is pretty good..
If you need any help with this.. Just let me know
